plugins/lookup/kustomize.py

```diff
@@ -0,0 +1,133 @@
+#
+#  Copyright 2021 Red Hat | Ansible
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = '''
+    lookup: kustomize
+
+    short_description: Build a set of kubernetes resources using a 'kustomization.yaml' file.
+
+    version_added: "1.0.0"
```

Comment:
```suggestion
    version_added: "2.2.0"
```
:rewind: :hourglass_flowing_sand:

Reply:
bad copy paste :-1: